src/main/java/controller/CarRacingGame.java

```diff
@@ -61,35 +63,20 @@ private Cars createCars() {
         return new Cars(names);
     }
 
-    private Round createRap() {
-        String rawRap = inputView.inputRap();
-        validateIsNull(rawRap);
-        int rap = convertToInt(rawRap);
-
-        return new Round(rap);
-    }
-
     private void validateIsNull(String input) {
         if (input == null) {
             throw new IllegalArgumentException(Exceptions.NULL_EXCEPTION.getMessage());
         }
     }
 
-    private int convertToInt(String rawRap) {
-        validateNumberFormat(rawRap);
-        return Integer.parseInt(rawRap);
-    }
+    private Round createRound() {
+        int round = inputView.inputRound();
 
-    private void validateNumberFormat(String rawRap) {
-        try {
-            Integer.parseInt(rawRap);
-        } catch (NumberFormatException e) {
-            throw new IllegalArgumentException(Exceptions.NUMBER_FORMAT_EXCEPTION.getMessage());
-        }
+        return new Round(round);
     }
 
     private void printWinners(Cars cars) {
-        List<String> winners = judge.findWinners(cars);
+        List<String> winners = Winners.findWinners(cars).getWinners();
         outputView.printWinners(winners);
```

Comment:
이런 저런 선언 방법이 있을 수 있죠 😃 

아래와 같이 선언한 것과 리니와 같이 선언한 것은 어떤 느낌의 차이가 있는지 생각해봐도 좋을 것 같아요!

```suggestion
        Winners winners = Winners.findWinners(cars);
        outputView.printWinners(winners.getWinners());
```

Reply:
타입으로 도메인을 사용하는 것이 더 가독성 있습니다! 타입만 확인했을 때 findWinners()가 List<String>을 반환한다고 혼란을 줄 수 있을 것 같아요.